common/lib/capa/capa/capa_problem.py

```diff
@@ -62,6 +74,9 @@
 # main class for this module
 
 
+DEFAULT_QUESTION_TEXT = _("You must specify meaningful question text.")
```

Comment:
I conferred with several others, and we propose:
"Formatting error: You have not explicitly identified the question text." 

If it is possible to include a link that renders correctly (I know that has been problematic in the simple editor) it would be nice to have a link to 

http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/course_components/create_problem.html#editing-a-problem-in-studio

worded something like "See the _edX documentation_ for more information.